### PR TITLE
Docs: Added steps to configure env variables on README.md

### LINKED ISSUE #642: Add setting up env file in README
## Description
As a newcomer to this project, I need to know how this project needs to be setup the easiest way.

#592 adds support for loading configuration variables through a `,env` file, but there is no corresponding
documentation for it.

@saideepesh000 I cannot seem to assign you but feel free to start working on this

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by saideepesh000**
### Description
As a developer, I need to know the steps to configure environment variables to run the project.

Fixes #642 

### Type of Change:

- Documentation

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented on my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

### CODE REVIEW

**Comment by saideepesh000** [comment]
@SanketDG, Please ignore the commits made up to "Update .travis.yml", I messed up with my develop branch, it is now ahead with the "Anitab-org/portal:develop" branch.

File: README.md
Comment (SanketDG) [inline README.md]: Just change this to:

```suggestion
1. Rename `.env.example` to `.env`.
```

**Review by SanketDG — DISMISSED** [review]
Just a few changes and I will merge this

Make sure to resolve the conflicts too!

**Comment by saideepesh000** [comment]
@SanketDG, Made the changes.

**Review by SanketDG — DISMISSED** [review]
Looks good! Great work, I will merge this soon as soon as I get another approval.

**Comment by saideepesh000** [comment]
> 
> 
> Looks good! Great work, I will merge this soon as soon as I get another approval.

Thank you :)

**Comment by saideepesh000** [comment]
> 
> 
> Looks good! Great work, I will merge this soon as soon as I get another approval.

Should I add "ZOOM_API_KEY, ZOOM_API_SECRET, ZOOM_USER_ID" or let it be a first-timer-issue

**Comment by SanketDG** [comment]
@saideepesh000 Feel free to add these, and also rebase and squash!

**Comment by saideepesh000** [comment]
> 
> 
> @saideepesh000 Feel free to add these, and also rebase and squash!

Okay

File: README.md
Comment (SanketDG) [inline README.md]:
```suggestion
    # External APIs (optional)
```

File: README.md
Comment (SanketDG) [inline README.md]:
```suggestion
    # Email settings (Optional)
```

**Review by SanketDG — DISMISSED** [review]
Just one tiny change and this should be good to go!

Make sure you also do the changes for the Windows instructions!

**Comment by SanketDG** [comment]
Can you also do for the Windows instructions? @saideepesh000

**Comment by saideepesh000** [comment]
> 
> 
> Can you also do for the Windows instructions? @saideepesh000

@SanketDG, which one ?, I added steps for both linux & windows, I am not getting you.

**Comment by saideepesh000** [comment]
> 
> 
> > Can you also do for the Windows instructions? @saideepesh000
> 
> @SanketDG, which one ?, I added steps for both linux & windows, I am not getting you.

Okay the "(Optional)", right?

**Review by SanketDG — APPROVED** [review]
Ready for Merge :tada: 

You have conflicts though :confused:

**Pull request merged**
